### DISCUSSION TITLE
CTD SOMETHING UNUSUAL FOR ME

### DISCUSSION THREAD

**Discussion by georgiosgiannoukos** [flybywiresim/aircraft#3463] [General]
Second CTD in two days with the A320 Dev mod during flight.

**Comment by ChruutvoLuzi** [comment]
Yes, there are a few quirk in the development version right now, hence the "Development" Version

Will get fixed for sure, patience

**Comment by hotshotp** [comment]
this isn't helpful, you should provide more info in a bug report rather than a discussion

**Comment by ruimirada** [comment]
The same happens to me, CTD constant but in my case always at the airport, without any information or error report, just close the simulator.
